coralproject/talk
# Add Toggleable component, update tab classes

## What does this PR do?
- adds Toggleable component to framework
- update tabbar with default classes.
 - change tab default classes to talk- from coral-

## How do I test this PR?
 - see new classes in tabs
 - Toggleable is only in use for off-topic code in development.
